SPECS/rubygem-parallel/rubygem-parallel.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
- "Signatures": {
-  "parallel-1.20.1.tar.gz": "fcc3c400a4a7cc38d7079592437d676160fa9b5e11d0e7224e622c7bbf110351"
- }
-}
+    "Signatures": {
```

Comment:
Can you revert the extra spaces, and make sure your editor keeps a newline at the end of files?

Reply:
In VSCODE the setting is called "Files: Insert Final Newline"

Reply:
removed spaces and using this Insert Final Newline